lib/SPIRV/OCLTypeToSPIRV.h

```diff
@@ -84,3 +87,5 @@ class OCLTypeToSPIRV : public ModulePass {
 };
 
 } // namespace SPIRV
+
+#endif
```

Comment:
Shouldn't it be 
`#endif // LLVM_TOOLS_SPIRV_LLVM_TRANSLATOR_LIB_SPIRV_OCLTYPETOSPIRV_H`
?

Reply:
I have no idea why clang-tidy didn’t automatically add the comment. I’ll manually add them.